avocado-core/src/main/scala/org/bdgenomics/avocado/algorithms/math/LogPhred.scala

```diff
@@ -0,0 +1,29 @@
+package org.bdgenomics.avocado.algorithms.math
```

Comment:
I remember you saying something about a linter/script to do this kind of functionality? Where can I find that?